<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-3IK4T82S" failures="3" tests="3" name="Scenarios Test" time="33.591" errors="0" timestamp="2025-02-26T14:32:25 EET">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.252">
    <failure type="java.lang.AssertionError" message="expected [Sign up successful.] but found [This user already exist.]">
      <![CDATA[java.lang.AssertionError: expected [Sign up successful.] but found [This user already exist.]
at Steps.TwoProductsPurchasing.the_user_has_registered_successfully_using(TwoProductsPurchasing.java:44)
at ✽.the user has registered successfully using "User4487" , "<password>"(file:///C:/Users/Mohamed%20Mazrouaa/IdeaProjects/VodafoneTask/src/test/java/features/ProductsPurchasing.feature:5)
... Removed 6 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.078">
    <failure type="java.lang.AssertionError" message="expected [Sign up successful.] but found [This user already exist.]">
      <![CDATA[java.lang.AssertionError: expected [Sign up successful.] but found [This user already exist.]
at Steps.UserRegisteration.the_sign_up_successful_message_appears_successfully(UserRegisteration.java:49)
at ✽.The “Sign up successful.” message appears successfully(file:///C:/Users/Mohamed%20Mazrouaa/IdeaProjects/VodafoneTask/src/test/java/features/UserRegisteration.feature:8)
... Removed 6 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.092">
    <failure type="java.lang.AssertionError" message="expected [Sign up successful.] but found [This user already exist.]">
      <![CDATA[java.lang.AssertionError: expected [Sign up successful.] but found [This user already exist.]
at Steps.UserRegisteration.the_sign_up_successful_message_appears_successfully(UserRegisteration.java:49)
at ✽.The “Sign up successful.” message appears successfully(file:///C:/Users/Mohamed%20Mazrouaa/IdeaProjects/VodafoneTask/src/test/java/features/UserRegisteration.feature:8)
... Removed 6 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Scenarios Test -->
